test/cppsuite/src/component/timestamp_manager.cpp

```diff
@@ -133,10 +133,10 @@ timestamp_manager::get_oldest_ts() const
 }
 
 wt_timestamp_t
-timestamp_manager::get_random_ts() const
+timestamp_manager::get_repeatable_read_ts() const
 {
     return random_generator::instance().generate_integer<wt_timestamp_t>(
-      _oldest_ts, static_cast<wt_timestamp_t>(get_time_now_s()));
+      _oldest_ts, _stable_ts - 1);
```

Comment:
You have to make sure `_stable_ts` and `_oldest_ts` are not the same (if they can be) if you do `_stable_ts - 1`.

Reply:
This should also guarantee that `_stable_ts` is more than `0`.

Reply:
Also, if `_stable_ts` is `0` and this returns `0`, we may run into the issue you wan to fix?

Reply:
I believe that would be a test configuration issue moreso than a framework issue.